### PR TITLE
Test add and remove

### PR DESCRIPTION
**Pull request by ShahierNashaat**
Pull request for reviewing unit testing for add and remove task using jest framework.

Test add new task function.

Add delete test.

Fix test delete task function.

### CODE REVIEW

File: src/add-and-remove-task.test.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+jest.mock('./localStorage-controller');
+
+describe('to do list add and delete', () => {
+  test('Add new task', () => {
+    const tasksArr = localStorageController.getDataFromLocalStorage('dum');
+    expect(taskController.addTask(`task${tasksArr.length + 1}`)).toHaveLength(3);
+  });
+  test('Delete task', () => {
+    expect(taskController.deleteTask(0)).toHaveLength(2);
+  });
```
Comment (dasileker) [inline src/add-and-remove-task.test.js:13]: - [ ] As the requirements say you should test all the pure functions, so you need to test also the edit, complete: true/false, and also clear all completed.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @ShahierNashaat,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/add-and-remove-task.test.js (hunk 1 above)
Comment (rloterh) [reply src/add-and-remove-task.test.js]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Shahier & Godanah , 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer(s). There is nothing else to say other than... it's time to merge it 🍾 :shipit:.
Congratulations! 🎉🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**